packages/ui/src/Dialog/index.tsx

```diff
@@ -217,27 +219,43 @@ const Content = <IconOnlyButtonGroupProps extends boolean | undefined>({
     <RadixDialog.Portal>
       <Overlay />
 
-      <DialogContent>
-        <TitleAndCloseButton>
-          <RadixDialog.Title asChild>
-            <Text xxl as='h2'>
-              {title}
-            </Text>
-          </RadixDialog.Title>
+      <RadixDialog.Content>
```

Comment:
This part is easier to review with "hide whitespace" enabled

Reply:
interesting. i tried this option on GitHub but didn't notice the changes. What does it do?

Reply:
In the diff view, it hides lines whose only changes are whitespace (i.e., indentation). It makes it easier to see what's changed in a file like this, where a new parent component was added to the component tree, thus resulting in indents to all the markup below it in the tree.